flatliners/comparisonscore.py

```diff
@@ -25,7 +25,9 @@ def on_next(self, x):
         if is_cluster_record:
             cluster_id = x['cluster']
             self.compute_cluster_distance(x)
-            self.publish(self.score[cluster_id])
+            if self.wait_for_resources(x):
```

Comment:
on naming:
I would expect `wait_for_resources` to be true, if I should wait for the resources. But it's the other way round.
You could either change to `if not self.wait_for_resources`
Or, rename the method, because "positive" is always better.
`if self.ready_to_publish(x)`

Reply:
Ok, will change name to `ready_to_publish()`